### ISSUE TITLE
Import xlsx into Google Sheets : text links are broken

### ISSUE THREAD

**Issue by JohnDoe8521** [PHPOffice/PhpSpreadsheet#807]
This is:

```
- [x ] a bug report
```

### What is the expected behavior?
Xlsx content (with internal links) generated and imported into Google Sheets (Google Drive) should be ok.
(conversion is ok (with internal links) if we create the file with MS Excel)

### What is the current behavior?
After the conversion by google Sheet, internal links text are replaced by "#gid=xxxx".
I tried to compare the xml files between the file generated by phpspreadsheet and another file created by MS Excel and the difference is on the attribute of hyperlink tag : there is "display" attribute in the MS Excel file. I tried setting tooltip but it completes tooltip attribute.
phpspreadsheet :  ```<hyperlink ref="B2" location="'My Data'!A1" tooltip="test"/>```
MS Excel :  ```<hyperlink ref="E26" location="'My Data'!A1" display="test"/>```

### What are the steps to reproduce?
```php
<?php

require __DIR__ . '/vendor/autoload.php';

// just create a new sheet and a link to that sheet
$spreadsheet = new \PhpOffice\PhpSpreadsheet\Spreadsheet();
$secondSheet= new \PhpOffice\PhpSpreadsheet\Worksheet\Worksheet($spreadsheet, 'My Data');
$spreadsheet->addSheet($secondSheet, 1);
$spreadsheet->getActiveSheet()->setCellValue('B2', 'test');
$spreadsheet->getActiveSheet()->getCell('B2')->getHyperlink()->setUrl("sheet://'My Data'!A1");
$spreadsheet->getActiveSheet()->getCell('B2')->getHyperlink()->setTooltip("test");
$spreadsheet->setActiveSheetIndex(0);
$helper->write($spreadsheet, "test.xlsx");
```
Import the generated xlsx into google drive and try to read the sheet : links are ids

### Which versions of PhpSpreadsheet and PHP are affected?
1.5.2, PHP 7.0

**Comment by JohnDoe8521** [comment]
To resolve this problem, I need to add this line : 
Worksheet.php l629
$objWriter->writeAttribute('display', $hyperlink->getTooltip());

**Comment by oleibman** [comment]
The supplied code seems to work okay in Google Sheets today, without the need to supply a display attribute. I do not find the case for it compelling (it would just take the place of the data in the cell, so you can just set the cell value to what you want). Removing stale label, leaving issue closed.

**Comment by oleibman** [comment]
Actually the `display` part seems to have been implemented (on write) by PR #904. I need to add equivalent code for read, which I will do as part of PR #4764.
